src/apps/chatter/rooms/index.js

```diff
@@ -1,36 +1,30 @@
 // We are going to use ListResource to fetch and display results of
 // an API request. You could also use the lower-level [withRequest HoC](https://github.com/inputlogic/elements/tree/master/components/with-request)
-import ListResource from '@app-elements/list-resource'
-
+import { useRequest } from '@app-elements/use-request'
+import LoadingIndicator from '@app-elements/loading-indicator'
 import { Link } from '@app-elements/router'
-
 import url from '/util/url'
-
-// Just some simple styles for this page
 import './rooms.less'
+import store from '/store'
 
-// We need to define a Component that represents each item returned
-// from the API request. In our case, the API response looks like:
-//   [
-//     { id, name, email },
-//     ...
-//   ]
-const RoomItem = ({ id, name, email }) =>
+const RoomItem = ({ id, name }) =>
   <div class='room-item'>
-    <h2><Link name='room' args={{ id }}>{name}</Link></h2>
-    <p>{email}</p>
+    <h3><Link name='room' args={{ id }}>{name}</Link></h3>
   </div>
 
-// In our page component we render a ListResource instance, telling it
-// the endpoint to fetch results from, to limit the results to `10`, and
-// to render each result item with our `UserItem` component.
-const Rooms = () =>
-  <div className='container pt-7 pb-4'>
-    <ListResource
-      endpoint={url('api.rooms')}
-      limit={10}
-      render={RoomItem}
-    />
-  </div>
+const Rooms = () => {
```

Comment:
Great work migrating to useRequest ✨ 

Unfortunately, the example code had a bad practice that I'm hoping to do away with: default exports.

I think it's better to use named exports exclusively as it prevents typos and ensures easy refactoring.

With default exports I could have a typo where I import `ChartRoom`. If I wanted to refactor/rename the component to just `Room`, a project-wide search for `ChatRoom` would miss my typo of `ChartRoom`.

In this case, can be updated to `export function Rooms () {...}`. And, your imports will have to change from `import Rooms from '...'` to `import { Rooms } from '...'`